cpp/cgi_progs/message_dispatcher.cc

```diff
@@ -66,6 +90,7 @@ int Main(int argc, char *argv[]) {
     // Wait for incoming message 
     while(true) {
        sd_bus_wait(bus, UINT64_MAX);
+       InitializeTimer();
```

Comment:
Must probably be called once before the while loop is entered. Otherwise it might never exit on itself.